examples/reference-entity-model/steps/mapping/mapCustomersWithRelatedEntitiesJSON.step.json

```diff
@@ -98,7 +98,6 @@
           "sourcedFrom": "SendToID"
         },
         "lineItems": {
-          "sourcedFrom": "Products",
```

Comment:
Why is this removed ?

Reply:
Thanks @srinathgit . That line should not have been removed. Will make the change